core/src/main/java/edu/wpi/grip/core/operations/opencv/NewPointOperation.java

```diff
@@ -42,6 +42,8 @@ public NewPointOperation(InputSocket.Factory inputSocketFactory, OutputSocket.Fa
     this.ySocket = inputSocketFactory.create(yHint);
 
     this.outputSocket = outputSocketFactory.create(outputHint);
+    
+    this.perform();//So initial value is correct.
```

Comment:
__DO NOT DO THIS__

Use a builder for the socket hint to define a default value.